platform/security/policy/security-policy-context/src/test/java/org/codice/security/policy/context/PolicyManagerTest.java

```diff
@@ -119,6 +124,37 @@ public void setup() {
         rollBackTestManager.setPolicies(contextPolicies);
     }
 
+    @Test
+    public void testBadTraversal() {
```

Comment:
Somewhat of a nit, but this could be difficult to debug a breaking change. A little commentary on these edge cases would help.

May also want to consider testing the `MAX_TRAVERSAL_DEPTH` upper bound.